libraries/extensions/dora-rerun/src/main.rs

```diff
@@ -0,0 +1,86 @@
+//! Demonstrates the most barebone usage of the Rerun SDK.
+
+use dora_node_api::{
+    arrow::{
+        array::{PrimitiveArray, StringArray, UInt8Array},
+        datatypes::{UInt8Type, Utf8Type},
+        ipc::Utf8,
+    },
+    DoraNode, Event,
+};
+use eyre::{Context, Result};
+use rerun::{
+    external::{arrow2::array::Utf8Array, re_types::ArrowBuffer},
+    TensorBuffer, TensorData, TensorDimension,
+};
+
+fn main() -> Result<()> {
+    // `serve()` requires to have a running Tokio runtime in the current context.
+    let rt = tokio::runtime::Runtime::new().expect("Failed to create tokio runtime");
+    let _guard = rt.enter();
+
+    let (_node, mut events) =
+        DoraNode::init_from_env().context("Could not initialize dora node")?;
+
+    let rec = rerun::RecordingStreamBuilder::new("dora-rerun")
+        .spawn()
+        .context("Could not spawn rerun visualization")?;
+
+    let shape = vec![
```

Comment:
These fields are in the wrong order and should be ordered: (height, width, depth) instead.

The name isn't actually used other than for informational purposes. So dim[0] will be height of image regardless of what it's called. In python logging I believe we have a validator that will generate a warning in this case, but seems to be missing on the rust side.

Reply:
Got it! Thanks so much. This has been fixed.